### PR TITLE
ES-259583 Add UG to remove default chart title from the Word document

### PR DESCRIPTION
**Pull request by AkashArul26**
Added UG to remove default chart title from the Word document

### CODE REVIEW

File: File-Formats/DocIO/Working-with-Charts.md [hunk 1]
```diff
@@ -759,6 +759,82 @@ document.Close()
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Charts/Applying-3D-formats).
 
+## Removing Chart Title
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Charts.md:762]: Remove Chart Title

File: File-Formats/DocIO/Working-with-Charts.md [hunk 2]
```diff
@@ -759,6 +759,82 @@ document.Close()
...
+
+The following code example illustrates how to remove the chart title from the document.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Charts.md:764]: The following code example illustrates how to remove the chart title from the Word document.

File: File-Formats/DocIO/Working-with-Charts.md [hunk 3]
```diff
@@ -759,6 +759,82 @@ document.Close()
...
+        chart.ChartData.SetValue(2, 2, 141.396);
+        chart.ChartData.SetValue(3, 1, "Stanley Hudson");
+        chart.ChartData.SetValue(3, 2, 80.368);
+        //Creates a new chart series with the name “Sales”.
+        IOfficeChartSerie pieSeries = chart.Series.Add("Sales");
+        pieSeries.Values = chart.ChartData[2, 2, 3, 2];
+        //Sets category labels.
+        chart.PrimaryCategoryAxis.CategoryLabels = chart.ChartData[2, 1, 3, 1];
+        //Creates file stream.
+        using (FileStream outputFileStream = new FileStream(Path.GetFullPath(@"../../../Result.docx"), FileMode.Create, FileAccess.ReadWrite))
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Charts.md:795]: Please use saving code as memory stream

//Saves the Word document to MemoryStream.
MemoryStream stream = new MemoryStream();
document.Save(stream, FormatType.Docx);
//Closes the document.
document.Close();

File: File-Formats/DocIO/Working-with-Charts.md [hunk 4]
```diff
@@ -759,6 +759,82 @@ document.Close()
...
+ chart.ChartData.SetValue(2, 2, 141.396)
+ chart.ChartData.SetValue(3, 1, "Stanley Hudson")
+ chart.ChartData.SetValue(3, 2, 80.368)
+'Creates a new chart series with the name “Sales”.
+ Dim pieSeries As IOfficeChartSerie =  chart.Series.Add("Sales") 
+ pieSeries.Values = chart.ChartData(2, 2, 3, 2)
+'Sets category labels.
+ chart.PrimaryCategoryAxis.CategoryLabels = chart.ChartData(2, 1, 3, 1)
+'Creates file stream.
+ Imports (FileStream outputFileStream = New FileStream(Path.GetFullPath("../../../Result.docx"), FileMode.Create, FileAccess.ReadWrite))
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Charts.md:827]: For VB, we can save directly in path. Please modify the code

File: File-Formats/DocIO/Working-with-Charts.md [hunk 5]
```diff
@@ -759,6 +759,82 @@ document.Close()
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Charts/Applying-3D-formats).
 
+## Removing Chart Title
+
+The following code example illustrates how to remove the chart title from the document.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C# [Cross-platform]" %}
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Charts.md:768]: Windows Platform code is missing

File: File-Formats/DocIO/Working-with-Charts.md [hunk 6]
```diff
@@ -759,6 +759,82 @@ document.Close()
...
+ Imports (FileStream outputFileStream = New FileStream(Path.GetFullPath("../../../Result.docx"), FileMode.Create, FileAccess.ReadWrite))
+ {
+     'Saves the Word document to file stream.
+      document.Save(outputFileStream, FormatType.Docx)
+ }
+{% endhighlight %}
+
+{% endtabs %}
+
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Charts/Remove-chart-title/.NET%20Standard)
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Charts.md:836]: Please use the link
https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Charts/Remove-chart-title

File: File-Formats/DocIO/Working-with-Charts.md (hunk 1 above)
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Charts.md:762]: We have separate page for Chart Title, 
As we mentioned in the task goals, please add as last topic in this page https://help.syncfusion.com/file-formats/docio/charts/chart-title

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Working-with-Charts.md (hunk 1 above)
Comment (AkashArul26) [reply File-Formats/DocIO/Working-with-Charts.md]: Addressed the feedback

File: File-Formats/DocIO/Working-with-Charts.md (hunk 2 above)
Comment (AkashArul26) [reply File-Formats/DocIO/Working-with-Charts.md]: Addressed the feedback

File: File-Formats/DocIO/Working-with-Charts.md (hunk 3 above)
Comment (AkashArul26) [reply File-Formats/DocIO/Working-with-Charts.md]: Addressed the feedback

File: File-Formats/DocIO/Working-with-Charts.md (hunk 4 above)
Comment (AkashArul26) [reply File-Formats/DocIO/Working-with-Charts.md]: Addressed the feedback

File: File-Formats/DocIO/Working-with-Charts.md (hunk 5 above)
Comment (AkashArul26) [reply File-Formats/DocIO/Working-with-Charts.md]: Addressed the feedback

File: File-Formats/DocIO/Working-with-Charts.md (hunk 6 above)
Comment (AkashArul26) [reply File-Formats/DocIO/Working-with-Charts.md]: Addressed the feedback

File: File-Formats/DocIO/Working-with-Charts.md (hunk 1 above)
Comment (AkashArul26) [reply File-Formats/DocIO/Working-with-Charts.md]: Addressed the feedback

File: File-Formats/DocIO/Charts/Chart-Title.md [hunk 7]
```diff
@@ -196,4 +196,109 @@ End Using
...
+        //Saves the Word document to MemoryStream.
+        MemoryStream stream = new MemoryStream();
+        document.Save(stream, FormatType.Docx);
+        //Closes the document.
+        document.Close();
+}
+{% endhighlight %}
+
+{% highlight c# tabtitle="C# [Windows-specific]" %}
+using (WordDocument document = new WordDocument("Template.docx", FormatType.Docx))
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Charts/Chart-Title.md:242]: Please change as creating Word document

File: File-Formats/DocIO/Charts/Chart-Title.md [hunk 8]
```diff
@@ -196,4 +196,109 @@ End Using
...
+    pieSeries.Values = chart.ChartData[2, 2, 3, 2];
+    //Sets category labels.
+    chart.PrimaryCategoryAxis.CategoryLabels = chart.ChartData[2, 1, 3, 1];
+    //Saves the document
+    document.Save("Sample.docx");
+}
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET [Windows-specific]" %}
+'Adds section to the document.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Charts/Chart-Title.md:272]: Some code snippet are missing. Please confirm does this code works properly

File: File-Formats/DocIO/Charts/Chart-Title.md [hunk 9]
```diff
@@ -196,4 +196,109 @@ End Using
...
+ chart.ChartData.SetValue(2, 2, 141.396)
+ chart.ChartData.SetValue(3, 1, "Stanley Hudson")
+ chart.ChartData.SetValue(3, 2, 80.368)
+'Creates a new chart series with the name “Sales”.
+ Dim pieSeries As IOfficeChartSerie =  chart.Series.Add("Sales") 
+ pieSeries.Values = chart.ChartData(2, 2, 3, 2)
+'Sets category labels.
+ chart.PrimaryCategoryAxis.CategoryLabels = chart.ChartData(2, 1, 3, 1)
+'Creates file stream.
+ Imports (MemoryStream stream = New MemoryStream())
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Charts/Chart-Title.md:295]: Please save in path, for VB as mentioned in earlier feedback.

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Charts/Chart-Title.md [hunk 10]
```diff
@@ -196,4 +196,109 @@ End Using
...
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Charts/Format-Chart-Title/.NET).
+
+## Remove Chart Title
+
+The following code example illustrates how to remove the chart title from the Word document.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+//Creates a new instance of WordDocument (Empty Word Document).
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Charts/Chart-Title.md:208]: //Creates a new instance of WordDocument.

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Charts/Chart-Title.md (hunk 8 above)
Comment (AkashArul26) [reply File-Formats/DocIO/Charts/Chart-Title.md:277]: Added them missed code. Ensured the code that it is working.

File: File-Formats/DocIO/Charts/Chart-Title.md (hunk 7 above)
Comment (AkashArul26) [reply File-Formats/DocIO/Charts/Chart-Title.md]: Addressed the feedback

File: File-Formats/DocIO/Charts/Chart-Title.md (hunk 9 above)
Comment (AkashArul26) [reply File-Formats/DocIO/Charts/Chart-Title.md]: Addressed the feedback

**Review by MohanaselvamJothi — APPROVED** [review]

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**